cypress-io/cypress
# chore: notification telemetry

- Closes #26785

### Additional details


Capture some non-intrusive and non identifying telemetry for App Notifications.

### Steps to test


There are two ways a user might navigate to the Debug page:

- via side navigation
- via a notification

We want to capture how the user is navigating to the Debug page. That's what you are testing here.

The easiest way to do this is to turn on debugging for `cypress:data-context:actions:EventCollectorActions` and look for a message that says `Recorded machine-linked` with the expected event info.

Then you can trigger the event like this:

1. `source: sidebar` - just click the Debug icon in from the App. It will launch a mutation that eventually calls the `recordEvent` action.
2. `source: notification`. This is a bit more tricky. You'll need to trigger a notification. You can see how to do that by looking at [this PRs test plan](https://github.com/cypress-io/cypress/pull/26974). Once you click a notification, it should also record the event with the correct `medium`.

Example:

<img width="1079" alt="image" src="https://github.com/cypress-io/cypress/assets/1702361/6da3917a-68f1-4706-a441-17bde3f58e18">


### How has the user experience changed?


### PR Tasks
